File: src/main/kotlin/com/cleitech/receipt/shoeboxed/domain/Attachment.java
package com.cleitech.receipt.shoeboxed.domain;

import java.net.URL;

/**
 * @author Pierrick Puimean-Chieze on 23-04-16.
 */
public class Attachment {

    private String name;
    private  URL url;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public URL getUrl() {
        return url;
    }

    public void setUrl(URL url) {
        this.url = url;
    }

    @Override
    public String toString() {
        return "Attachment{" +
                "name='" + name + '\'' +
                ", url=" + url +
                '}';
    }
}

File: src/main/kotlin/com/cleitech/receipt/shoeboxed/domain/Attachment.kt
package com.cleitech.receipt.shoeboxed.domain

import java.net.URL

/**
 * @author Pierrick Puimean-Chieze on 23-04-16.
 */
class Attachment {

    var name: String? = null
    var url: URL? = null

    override fun toString(): String {
        return "Attachment{" +
                "name='" + name + '\''.toString() +
                ", url=" + url +
                '}'.toString()
    }
}
